packages/core/admin/admin/src/content-manager/components/BlocksEditor/hooks/tests/useBlocksStore.test.js

```diff
@@ -556,4 +556,60 @@ describe('useBlocksStore', () => {
       },
     ]);
   });
+
+  it('handles enter key called twice on paragraph block', () => {
+    const { result } = renderHook(useBlocksStore);
+
+    baseEditor.children = [
+      {
+        type: 'paragraph',
+        children: [
+          {
+            type: 'text',
+            text: 'Line of text',
+          },
+          {
+            type: 'text',
+            text: ' with modifiers too',
+            bold: true,
+          },
+        ],
+      },
+    ];
+
+    // Set the cursor after "Line of"
+    Transforms.select(baseEditor, {
+      /**
+       * Docs about anchor and focus: https://docs.slatejs.org/v/v0.47/slate-core/range
+       * In this case they are the same because no text is selected,
+       * we're only setting the position of the cursor.
+       */
+      anchor: Editor.point(baseEditor, { path: [0, 0], offset: 7 }),
+      focus: Editor.point(baseEditor, { path: [0, 0], offset: 7 }),
```

Comment:
Can you directly use, 
{ 
  anchor: { path: [0, 0], offset: 7 }),
  focus: { path: [0, 0], offset: 7 }),
 }
 
 No need of comment as well as its becoming more generic thing throughout the editor.